Added Licence field to bulk-edit form on Backoffice Asset.

xliiv opened the pull request:
Adding licence field to asset's bulk edit form is hard using regualar Django
mechanism. This pull request fixes that.

mkurek commented on src/ralph/back_office/admin.py:
```diff
@@ -115,7 +115,7 @@ class BackOfficeAssetAdmin(
     resource_class = resources.BackOfficeAssetResource
     bulk_edit_list = [
-        'status', 'barcode', 'imei', 'hostname', 'model', 'purchase_order',
+        'licence', 'status', 'barcode', 'imei', 'hostname', 'model', 'purchase_order',
```
licences

mkurek commented on src/ralph/back_office/admin.py:
```diff
@@ -151,6 +151,67 @@ class BackOfficeAssetAdmin(
+        from django.db.models.loading import get_model
+        Form = super().get_changelist_form(request, **kwargs)
+        class NewForm(Form):
+            licence = forms.ModelMultipleChoiceField(
```
licences

mkurek commented on src/ralph/back_office/admin.py:
```diff
@@ -151,6 +151,67 @@ class BackOfficeAssetAdmin(
+                pass
+
+            def save(self, commit=True):
+                form_licences = self.cleaned_data['licence']
```
move this logic to `save_m2m` (and call super there)

Commit:
Added Licence field to bulk-edit form on Backoffice Asset.

Adding licence field to asset's bulk edit form is hard using regualar Django
mechanism. This pull request fixes that.

mkurek commented:
Could you leave some `TODO: permissions handling` in your code?

ghost commented:
done

mkurek commented:
:ok:

szok commented:
:ok:
